shared/AppInsightsCommon/src/Offline.ts

```diff
@@ -53,21 +70,20 @@ export function createOfflineListener(parentEvtNamespace?: string | string[]): I
     function _enableEvents(target: any): boolean {
         let enabled = false;
         if (target) {
-            enabled = eventOn(target, "online", _setOnline, _evtNamespace);
+            enabled = eventOn(target, "online", _setOnlineState(true), _evtNamespace);
```

Comment:
No, there are 2 different states that this class is going to monitor
- runtime state (which is what these events are setting)
- user state (which is what will be set via the `setOnlineState`

Also you are "calling" the `_setOnlineState` here and not "passing a function.